### PR TITLE
Add backport config and update readme

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

### CODE REVIEW

**Comment by a-mccarthy** [comment]
@jonasrosland can you review this please? I was also think that it might be helpful to add a PR template so we can (hopefully) capture which branches changes should be backported into. Or maybe add labels to PRs? thoughts?

File: README.md [hunk 1]
```diff
@@ -8,16 +8,23 @@ The website is built and developed using the [Hugo](https://gohugo.io/) static s
...
-### Specific instructions for documentation
+The latest (edge) version of the documentation lives in the [`docs`](./docs) folder on the `master` branch, and is always viewable on https://goharbor.io/docs/edge. Documentation for specific released versions lives in its own release branch, for example `release-2.1.0`.
+
+### Creating a PR
+
+In general, all PRs should be made against the `master` branch to update the edge version of the docs. If you are making a change that also affects released versions, indicate which release branches to update in your PR so the website maintainers can backport your changes.
 
-The latest (edge) version of the documentation lives in the [`docs`](./docs) folder, and is always viewable on https://goharbor.io/docs/edge.
-When updating this version and making a PR, the edge docs will be viewable through the Netlify preview linked in the PR tests, just make sure you add `/docs/edge/` to the URL. That way you can verify that your changes look good before asking the maintainers for a review.
+If you are making a change that is specific to a single released version of documentation, make a PR against that branch (`release-X`).
```
Comment (jonasrosland) [inline README.md:23]: Can we clarify that this should only be done if the content isn't going to be used for any future versions? Like, this is a bugfix in the docs for 1.10, that's ok, but a small fix for 2.1 should most likly go into master/main and then backported, right?

File: README.md [hunk 2]
```diff
@@ -29,6 +36,8 @@ helmversion = "1.3"
 branchname = "release-X.Y.Z"
 ```
 
+After a releases, update the `.backportrc.json` file with the new release branch.
```
Comment (jonasrosland) [inline README.md:39]: Missing some words :)

**Review by jonasrosland — CHANGES_REQUESTED** [review]
I think this looks great, just 2 small edits needed IMO

**Comment by jonasrosland** [comment]
> @jonasrosland can you review this please? I was also think that it might be helpful to add a PR template so we can (hopefully) capture which branches changes should be backported into. Or maybe add labels to PRs? thoughts?

I think adding labels would make it clear when we do a glance over the new PRs, and adding a new PR template is a good idea too.

**Review by jonasrosland — APPROVED** [review]

**Pull request merged**